src/main/java/bq_standard/tasks/TaskRetrieval.java

```diff
@@ -341,8 +344,16 @@ public void run(ItemStack stack, IntFunction<ItemStack> consumer) {
                     int remaining = rStack.stackSize - value.getSecond()[i];
 
                     if (task.consume) {
-                        ItemStack removed = consumer.apply(remaining);
-                        value.getSecond()[i] += removed.stackSize;
+                        QuestingAPI.getLogger().error(task.getUnlocalisedName() + ", RUNNER: " + runner + ", FIRST: " + value.getFirst());
+                        if (QBConfig.fullySyncQuests && runner.equals(value.getFirst())){
+                            QuestingAPI.getLogger().error("RUN THROUGH");
```

Comment:
I think these logs were added for debugging purposes. If so, please remove ones.